MovieQuiz/Presentation/AlertPresenter.swift

```diff
@@ -8,10 +8,10 @@
 import Foundation
 import UIKit
 
-struct AlertPresenter: AlertPresenterProtocol {
+class AlertPresenter: AlertPresenterProtocol {
     
-    weak private var alertController: UIViewController?
-    init(alertController: UIViewController? = nil) {
+    private weak var alertController: UIViewController?
+    init(alertController: UIViewController) {
```

Comment:
Давай договоримся, между логическими блоками оставлять одну пустую строку. Например между функциями 
```suggestion
    private weak var alertController: UIViewController?
    
    init(alertController: UIViewController) {
```